src/components/backend-ai-settings-view.ts

```diff
@@ -320,6 +329,26 @@ export default class BackendAiSettingsView extends BackendAIPage {
                       </mwc-select>
                     </div>
                   </div>
+                  <div class="horizontal layout setting-item">
```

Comment:
How about add an icon and show dialog instead of exposing two inputs with descriptions?
(e.g. SSH configuration in usersettings menu)